bigbluebutton-html5/imports/ui/components/user-list/service.js

```diff
@@ -210,8 +211,23 @@ const getCurrentUser = () => {
   return (currentUser) ? mapUser(currentUser) : null;
 };
 
+const isMeetingLocked = () => {
+  const meeting = Meetings.findOne({});
```

Comment:
Not sure about a breakout room, but it definitely can be another meeting running in parallel.
And since Gabriel mentioned it - could you also change it [here](https://github.com/KDSBrowne/bigbluebutton/blob/c14e779bb453708de241b078d760f347ab7cfa71/bigbluebutton-html5/imports/ui/components/user-list/container.jsx#L41)?

Reply:
Double checked, we restrict that in publishers, so it can only be the case with breakout rooms.